lightning/src/ln/outbound_payment.rs

```diff
@@ -1868,6 +1898,7 @@ impl_writeable_tlv_based_enum_upgradable!(PendingOutboundPayment,
 		(0, expiration, required),
 		(2, retry_strategy, required),
 		(4, max_total_routing_fee_msat, option),
+		(6, invoice_request, option),
```

Comment:
Makes sense! Changes the tlv number to `5` so that downgrade will be supported in [c8c3349](https://github.com/lightningdevkit/rust-lightning/commit/c8c3349ce903f7bc0ff1d6267c4498caf5c5a59e)
Thanks!